.github/workflows/master_build_pr.yml

```diff
@@ -665,6 +662,8 @@ jobs:
       DOCKERHUB_ORG: ${{vars.DOCKERHUB_ORG}}
       GITHUB_TOKEN: ${{ secrets.CIPIPELINE_GITHUB_TOKEN }}
       GITHUB_USER: ${{ secrets.CIPIPELINE_GITHUB_USER }}
+      GITHUB_BASE_REF: ${GITHUB_BASE_REF}
```

Comment:
Where is the declaration?

Reply:
couldn't find a clear justification of how is it getting set, so used github.base_ref here